### PR TITLE
Bug 1237313 - Add API stability levels for hooks and fix typo

### PR DESCRIPTION
**Pull request by anarute**

### CODE REVIEW

File: routes/v1.js [hunk 1]
```diff
@@ -156,7 +160,7 @@ api.declare({
   name:         'getHookSchedule',
   output:       'hook-schedule.json',
   title:        'Get hook schedule',
-  stability:    'deprecated',
+  stability:    'experimental',
```
Comment (djmitche) [inline routes/v1.js]: This one should stay at "deprecated" - the information it provides is now available from `getHookStatus`.

File: test/api_test.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ suite('API', function() {
   }
 
   suite("createHook", function() {
-    test("creates a hookd", async () => {
+    test("creates a hook", async () => {
```
Comment (djmitche) [inline test/api_test.js]: good catch, thanks

**Comment by djmitche** [comment]
If you don't mind reverting that to "deprecated', I'll get this merged.

**Comment by anarute** [comment]
I reverted the getHookSchedule stability :)

**Comment by djmitche** [comment]
OK, just waiting for npm to recover so I can run the tests, and I'll get it
merged.  Grr..

**Pull request merged**